### DISCUSSION TITLE
Import Route from One Drive in .kurviger format

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#634] [Q&A] [answered]
Good morning,

When I select a .kurviger file in One Drive and want to import it with Cruiser, there is currently no selection in which format. Is it possible to define a setting in Cruiser 5.0 for the future in which format the import should take place by default? My current version is Cruiser 4.0.2
1st step: I select the .kurviger file in One Drive
2nd step: open with Cruiser
3rd step: now comes the query Overlay + Routing; Automatic; here the query which format is missing; 
4th step: I say Okay and now get the message invalid format

Guten Morgen,

wenn ich im One Drive eine .kurviger Datei auswähle und mit Cruiser importieren möchte, fehlt aktuell die Auswahl in welchem Format. Könnte man für die Zukunft im Cruiser 5.0 evtl. eine Einstellung hinterlegen in welchem Format der Import im Standard stattfinden soll? Meine Derzeitige Version ist die Cruiser 4.0.2
1.Schritt: ich wähle im One Drive die .kurviger Datei aus
2.Schritt: öffnen mit Cruiser
3.Schritt: jetzt kommt die Abfrage Overlay + Routing; Automatisch; hier fehlt die Abfrage welches Format; 
4:Schritt: ich sage Okay und bekomme nämlich jetzt die Meldung ungültiges Format

**Comment by devemux86** [comment]
@ToToGSARider 

This is automatic: click a route file with any file manager and select Cruiser to open it.
Do you use a file manager, like [Google Files](https://play.google.com/store/apps/details?id=com.google.android.apps.nbu.files)? Is the file valid?
Can you post the route file here to check it?

Dies geschieht automatisch: Klicken Sie mit einem beliebigen Dateimanager auf eine Routendatei und wählen Sie Cruiser, um sie zu öffnen.
Verwenden Sie einen Dateimanager wie [Google Files](https://play.google.com/store/apps/details?id=com.google.android.apps.nbu.files)? Ist die Datei gültig?
Können Sie die Routendatei hier posten, um sie zu überprüfen?

**Comment by ToToGSARider** [comment]
I have a Samsung S24+ and go to OneDrive and select the file there. If I want to open the file from Cruiser, it works without any problems. Mmhh, strange. I can't drag the file in .kurviger here. I have converted it to .zip.

Ich habe ein Samsung S24+ und gehe in den OneDrive und wähle dort die Datei aus. Wenn ich die Datei aus Cruiser heraus öffnen möchte geht es ohne Probleme. Mmhh, komisch. Ich kann die Datei im .kurviger nicht hier rein ziehen. ich habe sie mal in .zip gewandelt.

[07.08.2024 Eifel.zip](https://github.com/user-attachments/files/17074875/07.08.2024.Eifel.zip)

**Reply by devemux86** [reply]
@ToToGSARider 

I tried the file with Google Files and Total Commander and it works correctly.
Try opening it with a file manager instead of the cloud app.

Ich habe die Datei mit Google Files und Total Commander ausprobiert und sie funktioniert einwandfrei.
Versuchen Sie, die Datei mit einem Dateimanager zu öffnen, anstatt mit der Cloud-App.

**Comment by ToToGSARider** [comment]
I select the file in OneDrive and Cruiser opens directly, as I said always import with Cruiser. Then I confirm and the message Invalid file appears, but in the past I was asked for the file type?

Ich wähle die Datei im OneDrive aus und Cruiser öffnet sich ja direkt, da ich gesagt habe immer mit Cruiser Importieren. Dann bestätige ich und dann kommt Ungültige Datei. 
Früher kam hier doch die Abfrage nach dem Dateityp?
![Screenshot_20240920_145239_Cruiser](https://github.com/user-attachments/assets/d82c966f-2126-4d06-8b60-1829d05f2ea1)

**Reply by devemux86** [reply]
@ToToGSARider 

OneDrive may not be transferring the file correctly. Try using a file manager.

Vielleicht überträgt OneDrive die Datei nicht korrekt. Versuchen Sie es mit einem Dateimanager.

**Reply by ToToGSARider** [reply]
Oh no, now I've gone into One Drive so that I can use the import function in Cruiser and now the other way no longer works from One Drive with Cruiser open 🤣 I admit defeat. So now I only do it from Cruiser with import from One Drive. You can't have everything. Then please WunderLINQ short press to the right and open menu for import that you can then get from there into the One Drive. It would be really cool if you could define a standard import. That would be my wish for Cruiser 5.0. Thank you very much anyway. 

Och nö, jetzt bin ich extra in den One Drive, dass ich im Cruiser das Importieren nutzen kann und jetzt geht der andere Weg nicht mehr aus One Drive mit Cruiser öffnen 🤣 ich gebe mich geschlagen. Dannn mache ich es jetzt nur noch aus dem Cruiser heraus mit importieren aus One Drive. Man kann nicht alles haben. Dann bitte WunderLINQ kurzer druck nach Rechts und Menü für importieren öffnen, dass man dann von dort in den One Drive kommt. Es wäre echt cool, wenn man einen Standard Import definieren könnte. Wäre dann mein Wunsch für Cruiser 5.0. Trotzdem vielen Lieben Dank.

**Comment by devemux86** [comment, marked as answer]
@ToToGSARider 

I tried Google Drive and it opens files correctly with Cruiser.
So it may be a problem with OneDrive.
Or try a file manager or the import function of Cruiser.

Ich habe Google Drive ausprobiert und es öffnet Dateien korrekt mit Cruiser.
Daher kann es ein Problem mit OneDrive sein.
Oder versuchen Sie einen Dateimanager oder die Importfunktion von Cruiser.

**Reply by ToToGSARider** [reply]
It's all good. As I said, for Cruiser 5 I would like to be able to open the Routing menu on the WunderLINQ with a short press to the right and then I go on there with the control wheel with right left and then here the long press as confirmation. Then I go to OneDrive and select the file there. Since Cruiser is set as the default, Cruiser opens directly. Now it would be good if Cruiser recognized .kurviger directly and the import is confirmed by the next long press. Now it would be cool if the question Start navigation appeared. Then you would have complete operation via WunderLINQ and control wheel.

Es ist alles gut. Wie gesagt, ich würde mir für Cruiser 5 wünschen, dass man beim WunderLINQ durch einen kurzen Druck nach rechts das Menü Routing öffnen kann und dann gehe ich dort mit dem Kontrollrad weiter mit rechts links und dann hier der lange Druck als Bestätigung. Dann in den OneDrive und dort wähle ich dann die Datei aus. Da Cruiser als Standard hinterlegt ist, öffnet sich ja jetzt direkt der Cruiser. Jetzt wäre es gut wenn Cruiser direkt .kurviger erkennt und durch den nächsten langen Druck wird der Import bestätigt. Jetzt wäre es cool, wenn die Frage käme Navigation starten. Dann hätte man eine komplette Bedienung via WunderLINQ und Kontrollrad.

**Reply by devemux86** [reply]
@ToToGSARider 

Why do all these steps with external devices while driving?
Simply preload the route on Cruiser and press the navigation button before driving.

Warum all diese Schritte mit externen Geräten während der Fahrt durchführen?
Laden Sie einfach die Route auf dem Cruiser vor und drücken Sie vor der Fahrt die Navigationstaste.

**Reply by ToToGSARider** [reply]
Because it would be very cool to do as much as possible with the WunderLINQ. A short press to the left could then, as already mentioned, open the menu which can be reached by pressing the top left. Then select the points by scrolling or moving. I know from the past, with the Daytona remote control still in the K1, what you are capable of. That's why I like the Cruiser app so much and why I prefer it to any other app. My wife is also enthusiastic about the app.We start the Cruiser from the WunderLINQ app and that's why it would be perfect.


Weil es sehr cool wäre so viel wie möglich mit dem WunderLINQ zu machen. Kurzer Druck nach links könnte ja dann, wie schon mal erwähnt, das Menü öffnen was man erreicht wenn man oben links drückt. Hier dann durch Rollen oder bewegen die Punkte auswählen. Ich weiss aus der Vergangenheit, mit der Daytona Fernbedienung noch in der K1, was ihr in der Lage seid zu leisten. Deshalb mag ich die Cruiser App so sehr und ziehe sie deshalb einer anderen App vor. Auch meine Frau ist begeistert von der App. Wir starten den Cruiser doch aus der WunderLINQ App heraus und deshalb wäre es eben perfekt.
